lib/pdf_fill/forms/va686c674.rb

```diff
@@ -0,0 +1,1833 @@
+                limit: 18,
+                question_num: 13,
+                question_suffix: 'C',
+                question_text: 'INFORMATION NEEDED TO ADD SPOUSE > LIVING WITH SPOUSE > CITY'
+              },
+              'state_code' => {
+                key: 'form1[0].#subform[17].CurrentMailingAddress_StateOrProvince[1]',
+                limit: 2,
+                question_num: 13,
+                question_suffix: 'D',
+                question_text: 'INFORMATION NEEDED TO ADD SPOUSE > LIVING WITH SPOUSE > STATE'
+              },
+              'country_name' => {
+                key: 'form1[0].#subform[17].CurrentMailingAddress_Country[1]',
+                limit: 2,
+                question_num: 13,
+                question_suffix: 'E',
+                question_text: 'INFORMATION NEEDED TO ADD SPOUSE > LIVING WITH SPOUSE > COUNTRY'
+              },
+              'zip_code' => {
+                'firstFive' => {
+                  key: 'form1[0].#subform[17].CurrentMailingAddress_ZIPOrPostalCode_FirstFiveNumbers[1]',
+                  limit: 5,
+                  question_num: 13,
+                  question_suffix: 'F',
+                  question_text: 'INFORMATION NEEDED TO ADD SPOUSE > LIVING WITH SPOUSE > ZIP FIRST 5'
+                }, # end of zip first 5
+                'lastFour' => {
+                  key: 'form1[0].#subform[17].CurrentMailingAddress_ZIPOrPostalCode_LastFourNumbers[1]',
+                  limit: 4,
+                  question_num: 13,
+                  question_suffix: 'G',
+                  question_text: 'INFORMATION NEEDED TO ADD SPOUSE > LIVING WITH SPOUSE > ZIP LAST 4'
+                } # end of zip last 4
+              } # end of zip
+            } # end of address
+          }, # end does_live_with_spouse
+          'veteran_marriage_history' => {
+            limit: 4,
+            first_key: 'full_name',
+            'full_name' => {
+              'first' => {
+                key: 'veteran.previousSpouse.firstName[%iterator%]',
+                limit: 12,
+                question_num: 14,
+                question_suffix: 'A',
+                question_text: 'PREVIOUS MARRIAGE HISTORY >  PREVIOUS SPOUSE FIRST NAME'
+              },
+              'middleInitial' => {
+                key: 'form1[0].#subform[18].CHILDMiddleInitial1[%iterator%]',
+                limit: 1,
+                question_num: 14,
+                question_suffix: 'B',
+                question_text: 'PREVIOUS MARRIAGE HISTORY >  PREVIOUS SPOUSE MIDDLE INITIAL'
+              },
+              'last' => {
+                key: 'form1[0].#subform[18].CHILDLastName[%iterator%]',
+                limit: 18,
+                question_num: 14,
+                question_suffix: 'C',
+                question_text: 'PREVIOUS MARRIAGE HISTORY >  PREVIOUS SPOUSE LAST NAME'
+              }
+              # 'suffix' => {
+              # @TODO
+              # } # end of suffix
```

Comment:
Do you still need this?

Reply:
This is due to some differences between the front end and paper form, we still need to make a determination on how to handle these use cases.  I left the TODO comments in there as markers for when a decision is made and this is revisited.  I've added comments to hopefully make this a bit more clear.